home/templates/question_detail.html

```diff
@@ -217,8 +217,8 @@ <h2 class="question-title">{{ question.title }}</h2>
             <div class="question-metadata">
                 <div class="metadata-group">
                     <div class="metadata-item view-count">
-                        <i class="icon-eye"></i>
-                        <span>{{ view_count }} بازدید</span>
+                        <span id="viewCountText">{{ view_count }} بازدید</span>
```

Comment:
remove "بازدید" text
```suggestion
                        <span id="viewCountText">{{ view_count }}</span>
```